app/src/main/java/com/termux/api/CallLogAPI.java

```diff
@@ -44,6 +44,7 @@ private static void getCallLogs(Context context, JsonWriter out, int offset, int
             int dateIndex = cur.getColumnIndex(CallLog.Calls.DATE);
             int durationIndex = cur.getColumnIndex(CallLog.Calls.DURATION);
             int callTypeIndex = cur.getColumnIndex(CallLog.Calls.TYPE);
+            int simTypeIndex = cur.getColumnIndex(Callog.Calls.PHONE_ACCOUNT_ID);
```

Comment:
Should probably be 

```
CallLog.Calls.PHONE_ACCOUNT_ID
```
instead of 
```
Callog.Calls.PHONE_ACCOUNT_ID
```

Reply:
Yes, you're right. Sorry for the mix-up.